docs/data/sdks/flutter/index.md

```diff
@@ -403,6 +403,16 @@ Append the following Amplitude-JavaScript snippet into `web/index.html` in you
 </script>
 ```
 
+## Compatibility Matrix
+
+The following matrix lists which version of Gradle and plugins are compatibility with which Amplitude_Flutter.
+For Gradle Version lower than v7.3.3, We recommend to use the Amplitude Flutter v3.10.1.
+
+|<div class="med-column">Amplitude Flutter</div>|Gradle|Android Gradle Plugin|Kotlin Gradle Plugin|
```

Comment:
Not sure whether I understand this correctly, are `Android Gradle Plugin` and `Kotlin Gradle Plugin` versions are the minimum supported version or suggested version with the (`Amplitude Flutter`, `Gradle`) pair?

Reply:
It's not the minimum version. 
I don't think there have clear docs to talk about the compatible versions for all of those. Especially the previous android gradle plugin and kotlin gradle plugin are super old. 
Just want to say using those paired versions will make sure there is no compatibility issue.

Reply:
Thanks for the details! I think maybe we just put the external links for the Android Gradle Plugin and Kotlin Gradle Plugin to avoid confusion?
- https://developer.android.com/studio/releases/gradle-plugin#updating-gradle
- https://docs.gradle.org/current/userguide/compatibility.html#kotlin